Directory.Packages.props

```diff
@@ -89,6 +91,7 @@
     <PackageVersion Include="xunit.extensibility.core" Version="$(xUnitVersion)" />
     <PackageVersion Include="xunit.extensibility.execution" Version="$(xUnitVersion)" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="$(xUnitRunnerVersion)" />
+    <PackageVersion Include="Verify.Xunit" Version="28.15.0" />
```

Comment:
this is the latest supported version here https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/flat2/verify.xunit/index.json not sure why we are missing the newer versions https://www.nuget.org/packages/Verify.Xunit/

Reply:
done! updated the verify version